src/WebClient.spec.js

```diff
@@ -191,6 +191,19 @@ describe('WebClient', function () {
       });
     });
 
+    it('should return platform error if 200 request is not ok', function () {
+      const scope = nock('https://slack.com')
+        .post(/api/)
+        .reply(200, { ok: false });
+      const client = new WebClient(token);
+      return client.apiCall('method')
+        .catch((error) => {
+          assert.propertyVal(error, 'code', 'slackclient_platform_error');
+          assert.propertyVal(error.data, 'ok', false);
```

Comment:
this works since you have indirectly checked the truthiness of `error` in the preceding line. but i think this ca be improved as `assert.nestedPropertyVal(error, 'data.ok', false);` since then the order of the assertions wouldn't be significant.

Reply:
taken care of in latest commit; changed line 202 to `assert.nestedPropertyVal`